### ISSUE TITLE
Command to download cached parts list is out of date

### ISSUE THREAD

**Issue by Randall-Scharpf** [yaqwsx/jlcparts#133]
The first command is currently listed as
`wget https://yaqwsx.github.io/jlcparts/data/cache.zip https://yaqwsx.github.io/jlcparts/data/cache.z0{1..8}`

To get it running, I had to run
`wget https://yaqwsx.github.io/jlcparts/data/cache.zip https://yaqwsx.github.io/jlcparts/data/cache.z0{1..9} https://yaqwsx.github.io/jlcparts/data/cache.zip https://yaqwsx.github.io/jlcparts/data/cache.z{10..15}`

It looks like the issue is that more files representing parts of the cache have been added as the data has grown, perhaps? Not totally sure, just know this fixed the error with extraction. It might make sense to have the command sequence download a list of URLs to fetch and then send those to wget so that changes to the count of cache parts don't require continual updates to the development instructions, but that would add a few commands to the setup process.

Looking forward to getting this running and making a few local mods that I need, cheers!

**Comment by yaqwsx** [comment]
Yeah, the instructions are outdated. Follow the CI workflow: https://github.com/yaqwsx/jlcparts/blob/master/.github/workflows/update_components.yaml

**Comment by yaqwsx** [comment]
Fixed
